publish/publish.proj

```diff
@@ -99,6 +99,22 @@
                    FinalizeContainer="Runtime/$(SharedFrameworkNugetVersion)"
                    ForcePublish="true"                  
                    Condition="'@(_MissingBlobNames)' == ''" />
+    <Error Condition="'$(ChecksumAzureAccessToken)' == ''" Text="Missing required property 'ChecksumAzureAccessToken'" />
+    <Error Condition="'$(ChecksumAzureAccountName)' == ''" Text="Missing required property 'ChecksumAzureAccountName'" />
+    <Error Condition="'$(ChecksumContainerName)' == ''" Text="Missing required property 'ChecksumContainerName'" />
+    <Message Importance="High" Text="Finalizing Checksums" />
+    <FinalizeBuild AccountName="$(ChecksumAzureAccountName)"
```

Comment:
Yup this is going to the checksums account. Reusing the task seemed ok to me since it essentially ensures that the checksum account will always have the layout as the actual publish output.